### DISCUSSION TITLE
Create personal theme

### DISCUSSION THREAD

**Discussion by witch144** [prasanthrangan/hyprdots#2013] [Q&A] [answered]
how can i create my own for myself besides the official ones?

**Comment by cyb3rgh0u1** [comment, marked as answer]
you can create your own theme by following the theming wiki provided in readme, though it may be outdated. 

for the latest updates and to explore community contributions, visit the [ HyDE Gallery](https://github.com/HyDE-Project/hyde-gallery)

**Reply by witch144** [reply]
thanks! i'll check out the theming wiki and keep an eye on the hye gallery.
